MAUI/DataGrid/Column-types.md

```diff
@@ -0,0 +1,916 @@
+orderID.CellPadding = new Thickness(10, 0, 0, 0);
+orderID.HeaderPadding = new Thickness(10, 0, 0, 0);
+
+{% endhighlight %}
+{% endtabs %}
+
+## DataGridTextColumn
+
+DataGridTextColumn inherits all the properties of DataGridColumn. It is used to host the textual content in the record cells. Each of the record cell displays text based on the `MappingName` that associates the column with a property in the data source.
+
+The following code example creates DataGridTextColumn:
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:DataGridTextColumn="OrderID" /> 
+{% endhighlight %}
+{% highlight c# %}
+dataGrid.Columns.Add(DataGridTextColumn() { MappingName = "OrderID" });
+{% endhighlight %}
+{% endtabs %}
+
+The following topics explain the customizations done in the DataGridTextColumn:
+
+### Formatting
+
+To format values displayed in the DataGridColumn, use the `DataGridColumn.Format` property.
+
+#### Format column using StringFormat
+
+Assign the format of string to the `DataGridColumn.Format` property based on the bound data type of the property, the DataGridColumn is associated to format the value. You can use different  `StringFormats` to customize values displayed in the record cells.
+
+To apply formatting for a DataGridTextColumn, follow the code example:
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid.Columns>
+    <syncfusion:DataGridTextColumn  MappingName="Freight" Format="C" />
+    <syncfusion:DataGridTextColumn  MappingName="ShippingDate" Format="dd/MM/yyyy" />
+</syncfusion:SfDataGrid.Columns> 
+{% endhighlight %}
+{% highlight c# %}
+dataGrid.Columns.Add (DataGridTextColumn() { 
+    MappingName = "Customer",
+    Format = "C"
+});
+
+dataGrid.Columns.Add (DataGridTextColumn() { 
+    MappingName = "ShippingDate",
+    Format = "dd/MM/yyyy"
+});
+{% endhighlight %}
+{% endtabs %}
+
+#### Format column using converter
+
+Using converter, set the format of the column.
+
+To set the format using converter, follow the code example:
+
+ {% highlight xaml%}
+ <sfgrid:SfDataGrid.Columns>
+    <sfgrid:DataGridTextColumn MappingName="Salary" DisplayBinding="{Binding Salary, Converter={StaticResource SummaryConverter}}" />
+</sfgrid:SfDataGrid.Columns>  
+ {% endhighlight%}
+
+ {% highlight C#%}
+
+public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+{
+        var formattedString = string.Format("$ {0}", value);
+        return formattedString;
+}
+
+ {% endhighlight%}
+
+N> For AutoGenerated columns formatting can be applied by handling the `SfDataGrid.AutoGeneratingColumn` event.
+
+#### Formatting DataGridTextColumn with different culture
+
+To apply different `CultureInfo` for DataGridColumns, use the `DataGridColumn.CultureInfo` property. Assign format of the string to this property. Based on the type of the property the column is associated to format the value. You can use different `StringFormats` to customize values displayed in the record cells.
```

Comment:
To apply different `CultureInfo` for DataGridColumns, use the `DataGridColumn.CultureInfo` property. Assign the format of the string to this property. Based on the type of property the column is associated with, format the value. You can use different `StringFormats` to customize values displayed in the record cells.